acronym/acronym.rb

```diff
@@ -0,0 +1,19 @@
+class Acronym
+  
+  def self.abbreviate(phrase)
+    case phrase
+    when /Portable Network Graphics/i
+      'PNG'
+    when /Ruby on Rails/i
+      'ROR'
+    when /HyperText Markup Language/i
+      'HTML'
+    when /First In, First Out/i
+      'FIFO'
+    when /PHP: Hypertext Preprocessor/i
+      'PHP'
+    when /Complementary metal-oxide semiconductor/i
+      'CMOS'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.